compatibility-test/tests/go/coverall_test.go

```diff
@@ -2,187 +2,193 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
```

Comment:
I'm not sure what happened here. At first I though it was tabs/spaces, but even when matching tabs/spaces, it was still showing diff. I guess it was newlines? :thinking: